src/utils/storage.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import LOCAL_STORE, { STARTED_KEY, LOCKED_KEY, CLOSED_KEY, IS_BACKUP_DONE_KEY, IS_HARDWARE_KEY } from '../storage';
+
+const storageUtils = {
+  migratePreviousLocalStorage() {
+    const accessData = LOCAL_STORE.getItem('wallet:accessData');
+
+    if (!accessData) {
+      return;
+    }
+
+    if (LOCAL_STORE.getItem('wallet:started')) {
```

Comment:
I feel it would be easier to read if we had a map with the old key name instead of the string literals here.

For example:
```js
if (LOCAL_STORE.getItem(OLD_STARTED_KEY) {
  LOCAL_STORE.setItem(
    STARTED_KEY,
    LOCAL_STORE.getItem(OLD_STARTED_KEY),
  );
  LOCAL_STORE.removeItem(OLD_STARTED_KEY);
}
```
The map could even be within the `migratePreviousLocalStorage` function, not to pollute the file scope with unnecessary data.

Reply:
Done